fdbserver/workloads/TenantEntryCacheWorkload.actor.cpp

```diff
@@ -305,8 +305,18 @@ struct TenantEntryCacheWorkload : TestWorkload {
 	}
 
 	std::string description() const override { return "TenantEntryCache"; }
+
+	// Disable all fault injections for this test. They sometimes
```

Comment:
Will this be similar to extending `TenantEntryCache.toml` to disable `buggify` ?

```
[configuration]
buggify = false
```

Reply:
buggify introduces different kinds of errors than fault injection. I think both are necessary for this test.

Reply:
I'm unaware of how function works? Also, the "fault injection" in question is simulation injecting failures OR some other infrastructure doing it?

Reply:
We plan to enable automatic failure injection in simulation tests, outside of the ones specified in specific test files. This is enabled on this branch:
https://github.com/apple/foundationdb/tree/split-failure-injection-workloads

The overridden function here is a mechanism to disable specific fault-injection workloads that may be incompatible with a workload. To get this workload to pass, I've passed the special value "all" which will disable all automatic fault injection.

Reply:
Thanks for adding more context. 

Nit: The code documentation isn't very helpful in understanding the context, is there a plan to write a `design doc` (in `foundationdb/design/<> folder)? If yes, may be linking the `fault injection` design might help better. 

_Just a thought_: if there are various of such tests which falls under such a criteria, it may be useful to have a `white-listing` test.ini file; if a test gets added to such a list, then `fault injection` should ignore injecting faults there. The scheme may achieve the same, but, without making code changes.